test/show.jl

```diff
@@ -764,3 +764,21 @@ end
 @test static_shown(Symbol("")) == "Symbol(\"\")"
 @test static_shown(Symbol("a/b")) == "Symbol(\"a/b\")"
 @test static_shown(Symbol("a-b")) == "Symbol(\"a-b\")"
+
+
+# Test @show
+fname = tempname()
+f = open(fname, "w")
```

Comment:
would be good to use do-blocks for some of these, including the redirection so they get put back when finished